lib/controller.js

```diff
@@ -651,9 +651,30 @@ controller.getWifiInfo = function(opts) {
 
 controller.connectToNetwork = function(opts) {
   opts.authorized = true;
-  return controller.standardTesselCommand(opts, function(tessel) {
-    return tessel.connectToNetwork(opts);
-  });
+  var ssid = opts.ssid;
+  var password = opts.password;
+  var security = opts.security;
+  var securityOptions = ['none', 'wep', 'psk', 'psk2', 'wpa', 'wpa2'];
+
+  return new Promise(function(resolve, reject) {
+      if (!ssid) {
+        return reject('Invalid credentials. Must set ssid');
```

Comment:
A more helpful message might be `'Invalid credentials: must provide SSID with the -n option'`.